src/biz/bokhorst/xprivacy/XActivityManager.java

```diff
@@ -80,7 +80,7 @@ public String getClassName() {
 							listHook.add(new XActivityManager(act, null, null));
 						else
 							listHook.add(new XActivityManager(act, PrivacyManager.cSystem, null));
-				} else
+				} else if (!server)
```

Comment:
I try to prevent negated condition for readability, but it really doesn't matter much here.